### PR TITLE
Small bit of clean up for md files

### PR DESCRIPTION
**Pull request by Robvio**
Fixing up a few oddities in the doco's markup

### CODE REVIEW

File: hugo/content/posts/about.md [hunk 1]
```diff
@@ -7,16 +7,36 @@ menu:
...
- * `yt` is VERY experimental - please use with care.
+ * `yt` is experimental - please use with care.
 
 ## Goals
 
- * Query yaml documents
+  * Query yaml documents
   * Support multi-document files (with `---` separator lines)
- * Manipulate yaml documents
+  * Manipulate yaml documents
```
Comment (laher) [inline hugo/content/posts/about.md]: This indentation was intentional. Maybe subheadings would have been better

File: hugo/content/posts/scripting.md [hunk 2]
```diff
@@ -6,17 +6,17 @@ toc: true
 
 `yt` uses the [text/template package](https://golang.org/pkg/text/template) from [Go](https://golang.org)'s standard library.
 
-_NOTE: although text/template is a templating language, and yt allows you to use it as such, the intention is more to use it as a scripting language. This allows your yaml to remain as valid yaml_
+___NOTE__: although text/template is a templating language, and yt allows you to use it as such, the intention is more to use it as a scripting language. This allows your yaml to remain as valid yaml_
```
Comment (laher) [inline hugo/content/posts/scripting.md]: why the multi underscores?

**Review by laher — COMMENTED** [review]
Looks great - just 2 things

File: hugo/content/posts/about.md (hunk 1 above)
Comment (Robvio) [reply hugo/content/posts/about.md]: Whoops sorry!

**Review by laher — APPROVED** [review]
Thanks Rob, Contributor No.1

**Pull request merged**